libs/libxx/libxx_new.cxx

```diff
@@ -88,3 +88,18 @@ FAR void *operator new(std::size_t nbytes)
 
   return alloc;
 }
+
+FAR void *operator new(std::size_t nbytes, FAR void *ptr)
+{
+
+#ifdef CONFIG_DEBUG_ERROR
+  if (ptr == nullptr)
+    {
+      _err("ERROR: Failed to placement new\n");
```

Comment:
What is placement? Failed allocation?

Reply:
placement new never fail, here has some explanation:
https://en.wikipedia.org/wiki/Placement_syntax